addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5428,12 +5436,12 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13144"
-msgid "Change Apple Remote mode?"
+msgid "Change Apple remote mode?"
```

Comment:
the name of this specific thing is `Apple Remote`: http://en.wikipedia.org/wiki/Apple_Remote